operators/multiclusterobservability/manifests/base/config/metrics_allowlist.yaml

```diff
@@ -300,3 +300,57 @@ data:
         expr: sum(rate(node_network_receive_drop_total{job=\"node-exporter\", device!=\"lo\"}[1m])) without(device)
       - record: instance:node_network_transmit_drop_excluding_lo:rate1m
         expr: sum(rate(node_network_transmit_drop_total{job=\"node-exporter\", device!=\"lo\"}[1m])) without(device)
+  metrics_rules.yaml: |
+    - name: NamespaceHighCPUUsage
+      expr: (sum(kube_resourcequota{resource=\"limits.cpu\", type=\"used\"}) by (cluster, namespace, resourcequota) / sum(kube_resourcequota{resource=\"limits.cpu\", type=\"hard\"}) by (cluster, namespace, resourcequota)) * 100 > 70
```

Comment:
cluster label is not required in expr

Reply:
Removed the cluster label.

Reply:
@marcolan018 are there any other fields that we would want to capture?

Reply:
no, I think current expr is good